code-challenges.js

```diff
@@ -0,0 +1,135 @@
+// ASSESSMENT 1: Coding Practical Questions
+
+// To run file: navigate to the correct directory in terminal, $ node code-challenges.js
+
+// Pseudo coding your process is required.
+
+// --------------------INSTRUCTOR EXAMPLE: Create a function that takes in two strings and determines which of the two strings has more characters. Use the two sets of test variables provided.
+
+// Pseudo code:
+
+// Set one:
+const fruit1 = "apple"
+const fruit2 = "banana"
+// Expected outcome: "banana"
+
+// Set two:
+const fruit3 = "cherry"
+const fruit4 = "kiwi"
+// Expected outcome: "cherry"
+
+// --------------------1) Create a function that takes in a number and determines if the given number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below.
+
+// const (tempsArray)  = [42, 350, 212]
+// let boilingPoint = "212"
```

Comment:
I love that you assigned "212" to a variable. Make sure you mention that in your pseudocode and how you will use that in your conditional statements. Needs a bit of a refactor but this is a good start.